tests/Helper/ObjectParserTest.php

```diff
@@ -178,4 +178,18 @@ public function testDisabledCache(): void
         $this->assertNotSame($rules, $parser->getRules());
         $this->assertArrayNotHasKey($cacheKey, $cacheProperty->getValue());
     }
+
+    public function testUninitializedProperty(): void
```

Comment:
Add also test for `(new Validator())->validate($objectWithUnitializedProperty)`

Reply:
Added